app/controllers/api/base_controller/parameters.rb

```diff
@@ -82,6 +82,9 @@ def parse_filter(filter, operators)
                         [filter_value.gsub(/^'|'$/, ''), str_method]
                       when /^".*"$/
                         [filter_value.gsub(/^"|"$/, ''), str_method]
+                      when /^\d{4}-\d{2}-\d{2}/
```

Comment:
@abellotti this does not inspire confidence, but is it good enough? Alternatively we could try to parse it to determine if it's a date/time. It might be best if we knew if we were expecting a datetime, but I don't think that information is available here, but I can look into that.